WebRoot/messages/ZaMsg_ja.properties

```
@@ -988,6 +988,8 @@ MSG_zimbraPrefReadReceiptsToAddress = \u95b2\u89a7\u78ba\u8a8d\u30e1\u30c3\u30bb
 LBL_zimbraPrefReadReceiptsToAddress = \u95b2\u89a7\u78ba\u8a8d\u30e1\u30c3\u30bb\u30fc\u30b8\u306e\u8fd4\u4fe1\u30a2\u30c9\u30ec\u30b9\uff1a 
 LBL_zimbraPrefMailSendReadReceipts = \u95b2\u89a7\u78ba\u8a8d\u3092\u9001\u4fe1\uff1a
 LBL_zimbraFeatureReadReceiptsEnabled = \u95b2\u89a7\u78ba\u8a8d\u3092\u6709\u52b9\u306b\u3059\u308b
+LBL_zimbraFeatureEnableMailRecall = \u30e1\u30fc\u30eb\u306e\u53d6\u308a\u6d88\u3057
+LBL_zimbraFeatureMailRecallInterval = \u30e1\u30fc\u30eb\u306e\u53d6\u308a\u6d88\u3057\u9593\u9694
```

```suggestion
LBL_zimbraFeatureMailRecallInterval = \u30e1\u30fc\u30eb\u53d6\u308a\u6d88\u3057\u306e\u671f\u9650
```
(memo)
`メール取り消しの期限`

---

@k-kato, Correction done.
Thanks for the inputs.

---

Thank you for fixing it.